### PR TITLE
Added SensorMsgImage_CvMat.hpp/.cpp

### PR DESCRIPTION
**Pull request by AntoBongio**
Implements #1 

NOTE:
- Check name convention
- Consider removing cv_bridge
- This implementation is constrained to the following sensor_msgs::image_encodings: {MONO8, BGR8, MONO16, BHR16, BGRA8, BGRA16}

### CODE REVIEW

File: src/SensorMsgsImage_CvMat.cpp [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    destination.header = source.header;
+    destination = *cv_bridge::CvImage(source.header, encoding, source.mat).toImageMsg();
+}
+
+void rclcpp::TypeAdapter<CustomCvMat, sensor_msgs::msg::Image>::convert_to_custom (const ros_message_type & source, custom_type & destination){
+    
+    std::cerr << "[SensorMsgsImage_CvMat Adapter] Conversion from message" << std::endl;
+
+    destination.header = source.header;
+    destination.mat = cv_bridge::toCvCopy(source, source.encoding)->image;
```
Comment (roncapat) [inline src/SensorMsgsImage_CvMat.cpp]: Forse qua si può fare "move" per evitare la copia dei dati dalla `->image` alla `destination.mat`

Comment (roncapat) [reply src/SensorMsgsImage_CvMat.cpp]: cv::Mat supporta la move in teoria (https://docs.opencv.org/4.x/d3/d63/classcv_1_1Mat.html#adcaeb226bafeff8a1b262f57f6ffbcc4)

**Review by roncapat — DISMISSED** [review]

File: src/SensorMsgsImage_CvMat.cpp (hunk 1 above)
Comment (AntoBongio) [reply src/SensorMsgsImage_CvMat.cpp]: Si, potevo farla. Ho sistemato.

**Comment by roncapat** [comment]
## Proposal:

### Structure names:

StampedPointCloud_PCL\
StampedImage_CV\
StampedOccupancyGrid_CV

### Headers:

SensorsMsgImage_CvMat.hpp\
NavigationOccupancyGrid_CvMat.hpp\
SensorMsgPointCloud2_PCLPointCloud.hpp

### Dummy headers:

They simply include above headers, grouped by support library used. The user should include these headers directly instead of the ones above.

CV.hpp\
PCL.hpp

**Pull request merged**